### PR TITLE
Sunflower Implementation Change

### PR DESCRIPTION
**Pull request by Kirmito**
Created a new abstract class called EconomyPlants which extends Plant.
This class is used for plants that generate points

Removed the sunflower counter in place of this new implementation

### CODE REVIEW

**Comment by Kirmito** [comment]
I forgot something

File: src/assets/Twin_Flower.java [hunk 1]
```diff
@@ -25,7 +25,7 @@ public Twin_Flower()	{
 	 * Returns points of a twin sunflower
 	 * @return 
 	 */
-	public static int getPoints() {
+	public int getPoints() {
```
Comment (dershao) [inline src/assets/Twin_Flower.java]: Similarily here.

File: src/assets/Flower.java [hunk 2]
```diff
@@ -27,7 +27,7 @@ public Flower()	{
 	 * Returns the number of Points a sunflower gives the player per turn
 	 * @return the points of a sunflower  
 	 */
-	public static int getPoints() {
+	public int getPoints() {
```
Comment (dershao) [inline src/assets/Flower.java]: So similarly here, you don't need to override `getPoints()`

File: src/assets/EconomyPlant.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package assets;
+
+public abstract class EconomyPlant extends Plant {
+
+	public static final int points = 10;
```
[the remark this answers is not in the document]
Comment (dershao) [reply src/assets/EconomyPlant.java]: This is actually important if you want to reuse the `getPoints()` method for subclasses

**Comment by dershao** [comment]
🌥 🌦 🌤 🌞 🌻

**Pull request merged**